### PR TITLE
expand CAP architecture section, add conceptual diagrams

### PR DESCRIPTION
**Pull request by cjschroder**

### CODE REVIEW

File: xml/cap_depl_about.xml [hunk 1]
```diff
@@ -225,5 +227,90 @@
...
    creating and managing your own.)
   </para>
+
+  <para>
+   The following figures illustrate the main components of &productname;.
+   <xref linkend="fig.cap.cloud-101"/> shows a comparison of the basic cloud
+   platforms: Infrastructure as a Service (IaaS), Container as a Service
+   (CaaS), Platform as a Service (Paas), and Software as a Service (SaaS).
+   &suse; &caasp; is a Container as a Service platform, and &productname; is
+   Platform as a Service.
```
Comment (btat) [inline xml/cap_depl_about.xml]: is a Platform as a Service

Comment (cjschroder) [reply xml/cap_depl_about.xml]: Thanks, I was trying to not say 'platform' many times in one sentence.

**Comment by btat** [comment]
I believe this PR resolves https://github.com/SUSE/doc-cap/issues/94, so I'm closing that.

**Comment by btat** [comment]
Created https://github.com/SUSE/doc-cap/issues/185 as the current internal services diagram does not include the updated role compositions.

**Pull request merged**